### PR TITLE
Feat(Forms/PhoneInput): Initial version of a phone number input component

### PR DESCRIPTION
**Pull request by RostiMelk**

### CODE REVIEW

**Comment by RostiMelk** [comment]
I set the Prettier printWidth to 120 chars, and had to run the formatter on all the files, that's the reason for "46 files changed". 
Check new dir added: `src/stories/Forms/PhoneInput` @amfestoy

File: src/stories/Forms/PhoneInput/PhoneInput.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
```
Comment (amfestoy) [inline src/stories/Forms/PhoneInput/PhoneInput.stories.tsx:1]: defined but nevert used useState

File: src/stories/Forms/PhoneInput/PhoneInput.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+export const Primary = {
+  args: {
+    id: 'phoneNumber',
+    geoLocate: true,
+  },
+};
+
+const Template = ({ alert, hint, id, isRequired }) => {
+  return (
+    <>
```
Comment (amfestoy) [inline src/stories/Forms/PhoneInput/PhoneInput.stories.tsx:19]: Nitpick, but you dont need the empty <>

File: src/stories/Forms/PhoneInput/index.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  getExample,
+} from 'awesome-phonenumber';
+import { GlobalInputProps } from '../../../shared/types';
+import { Select, Input, Flex } from '../../../';
+
+interface WrapperProps {
+  hasError?: boolean;
+}
+
+const Wrapper = styled(Flex)<WrapperProps>`
```
Comment (amfestoy) [inline src/stories/Forms/PhoneInput/index.tsx:18]: Sometimes when it is just one prop like this I can think it is easier to read to just have it in curly brackets inside. So styled(Flex)<{hasError?: booleam}>. But no need to change this just a random thought while reading:)

File: src/stories/Forms/PhoneInput/index.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    [regionCode]
+  );
+
+  useEffect(
+    function handleOnChange(): void {
+      if (typeof onChange !== 'function') return;
+      const pn = parsePhoneNumber(phoneNumber, regionCode);
+      const json = pn.toJSON();
+      const value = pn.getNumber();
+      onChange({ target: { ...json, id, name: inputName, value } });
```
Comment (amfestoy) [inline src/stories/Forms/PhoneInput/index.tsx:119]: add comment here about event

**Comment by RostiMelk** [comment]
Merged into #44. Closing this.

**Pull request closed without merging**